aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/CreateHandler.java

```diff
@@ -163,7 +163,7 @@ protected ProgressEvent<ResourceModel, CallbackContext> modifyDBCluster(
             final ProgressEvent<ResourceModel, CallbackContext> progress
     ) {
         return proxy.initiate("rds::modify-dbcluster", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
-                .translateToServiceRequest(Translator::modifyDbClusterRequest)
+                .translateToServiceRequest(model -> Translator.modifyDbClusterRequest(model, model, false))
```

Comment:
should be `Translator.modifyDbClusterRequest(previousModel, desiredModel, false))` instead of `Translator.modifyDbClusterRequest(model, model, false))`